extensions/amp-analytics/0.1/vendors.js

```diff
@@ -394,6 +394,64 @@ export const ANALYTICS_CONFIG = /** @type {!JSONType} */ ({
     },
   },
 
+  'dynatrace': {
+    'requests': {
+      'endpoint': '${protocol}://${tenant}.${environment}:${port}/ampbf',
+      'pageview': '${endpoint}?type=js&' +
+        'flavor=amp&' +
+        'v=1&' +
+        'a=1%7C1%7C_load_%7C_load_%7C-%7C${navTiming(navigationStart)}%7C${navTiming(domContentLoadedEventEnd)}%7C0%2C2%7C2%7C_onload_%7C_load_%7C-%7C${navTiming(domContentLoadedEventStart)}%7C${navTiming(domContentLoadedEventEnd)}%7C0&' +
+        'fId=${pageViewId}&' +
+        'vID=${clientId(rxVisitor)}&' +
+        'referer=${sourceUrl}&' +
+        'title=${title}&' +
+        'sw=${screenWidth}&' +
+        'sh=${screenHeight}&' +
+        'w=${viewportWidth}&' +
+        'h=${viewportHeight}&' +
+        'nt=a${navType}' +
+        'b${navTiming(navigationStart)}' +
+        'c${navTiming(navigationStart,redirectStart)}' +
+        'd${navTiming(navigationStart,redirectEnd)}' +
+        'e${navTiming(navigationStart,fetchStart)}' +
+        'f${navTiming(navigationStart,domainLookupStart)}' +
+        'g${navTiming(navigationStart,domainLookupEnd)}' +
+        'h${navTiming(navigationStart,connectStart)}' +
+        'i${navTiming(navigationStart,connectEnd)}' +
+        'j${navTiming(navigationStart,secureConnectionStart)}' +
+        'k${navTiming(navigationStart,requestStart)}' +
+        'l${navTiming(navigationStart,responseStart)}' +
+        'm${navTiming(navigationStart,responseEnd)}' +
+        'n${navTiming(navigationStart,domLoading)}' +
+        'o${navTiming(navigationStart,domInteractive)}' +
+        'p${navTiming(navigationStart,domContentLoadedEventStart)}' +
+        'q${navTiming(navigationStart,domContentLoadedEventEnd)}' +
+        'r${navTiming(navigationStart,domComplete)}' +
+        's${navTiming(navigationStart,loadEventStart)}' +
+        't${navTiming(navigationStart,loadEventEnd)}&' +
+        'app=${app}&' +
+        'time=${timestamp}'
+    },
+    'triggers': {
+      'trackPageview': {
+        'on': 'visible',
+        'request': 'pageview'
+      }
+    },
+    'transport': {
+      'beacon': false,
```

Comment:
This is intentional. I have fixed the merge conflict(at least i think so, i am quite new to Git)